_vector-search/optimizing-performance/knn-vector-quantization.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: default
+title: Vector quantization
+parent: Optimizing vector search performance
+nav_order: 10
+has_children: true
+has_toc: false
+redirect_from:
+  - /search-plugins/knn/knn-vector-quantization/
+outside_cards:
+  - heading: "Byte vectors"
```

Comment:
Need to add a card for binary vectors along with byte vectors
https://opensearch.org/docs/latest/field-types/supported-field-types/knn-vector#binary-vectors

Reply:
@naveentatikonda Thanks! I addressed both comments. Could you review this commit https://github.com/opensearch-project/documentation-website/pull/9135/commits/a5e8b8db410d5d6538bc88abd5df5d82827d3494.

Reply:
@kolchfa-aws changes looks good. Thanks for making those changes. 

For binary vectors, we also need to add memory estimation. Here is the formula for HNSW
`1.1 * (dimension / 8 + 8 * M) bytes/vector`

For IVF, I guess it is `1.1 * (((dimension / 8) * num_vectors) + (nlist * dimension / 8))`. @jmazanec15 can you pls confirm ?

Reply:
Yeah that looks good to me

Reply:
Added both formulas.